examples/pong.rs

```diff
@@ -11,7 +11,8 @@ const PAD_HEIGHT: i32 = 3;
 
 struct PlayerState {
     pub position: Vec2,
-    pub direction: i32, // -1, 0, 1
+    pub direction: i32,
+    // -1, 0, 1
```

Comment:
This comment should apply to the direction. It was clippy automatically, I think.
```suggestion
    // -1, 0, 1
    pub direction: i32,
```

Reply:
Probably!